actions/src/queryopenstack/classes/list_ips.py

```diff
@@ -34,10 +34,16 @@ def __init__(self, conn):
             "not_project_id": lambda func_dict, args: func_dict["project_id"]
             not in args,
             "older_than": lambda func_dict, args: self.is_older_than_x_days(
-                func_dict, days=args
+                func_dict["created_at"], days=args[0]
```

Comment:
This is the old API and will not work for you, you'll need to write them using the Openstack API layer similar to https://github.com/DavidFair/st2-openstack-pack/blob/master/actions/src/project_action.py